tornado/platform/asyncio.py

```diff
@@ -258,10 +258,10 @@ def run_in_executor(
         func: Callable[..., _T],
         *args: Any
     ) -> Awaitable[_T]:
-        return self.asyncio_loop.run_in_executor(executor, func, *args)
+        return super().run_in_executor(executor, func, *args)
```

Comment:
The use of self.asyncio_loop was deliberate - the intention was to share a thread pool with the asyncio loop so that the base IOLoop.run_in_executor would never be run (so it's dead code and should be deleted). If we just wanted to call the super() methods, we don't even need to override run_in_executor and set_default_executor in AsyncIOLoop. 

Did you find a situation in which that was not the case and the un-prefixed threadpool from IOLoop.run_in_executor was being created?